app/src/main/java/com/infomaniak/mail/ui/main/thread/ThreadFragment.kt

```diff
@@ -117,12 +118,16 @@ class ThreadFragment : Fragment() {
 
         val defaultTextColor = context.getColor(R.color.primaryTextColor)
         appBar.addOnOffsetChangedListener { appBarLayout, verticalOffset ->
-            val total = appBarLayout.height * COLLAPSE_TITLE_THRESHOLD
-            val removed = appBarLayout.height - total
-            val progress = (((-verticalOffset.toFloat()) - removed).coerceAtLeast(0.0) / total).toFloat() // Between 0 and 1
-            val opacity = (progress * 255).roundToInt()
 
+            val subjectHeight = appBarLayout.height.toDouble()
```

Comment:
This `toDouble()` is not needed as both other lines will still be double due to computation always involving a double. Do we really want it?

Reply:
For the sake of clarity, I prefer to write it this way.